File: public/usage-examples/animations/create_animation_from_index_with_sound-1-example.py
from splashkit import *

script = load_animation_script("WalkingScript", "kermit.txt")

# Create animation from the script using index 0 and play sound if available
anim = create_animation_from_index_with_sound(script, 0, True)
write_line(f"Created animation from index 0 -> name: {animation_name(anim)}")

free_animation(anim)
free_animation_script(script)

File: public/usage-examples/animations/create_animation_from_index_with_sound-1-example.cpp
#include "splashkit.h"

int main()
{
    animation_script script = load_animation_script("WalkingScript", "kermit.txt");

    // Create an animation from index 0, play starting-frame sound if present
    animation anim = create_animation(script, 0, true);
    write_line("Created animation from index 0 -> name: %s", animation_name(anim).c_str());

    free_animation(anim);
    free_animation_script(script);
    return 0;
}
